### ISSUE TITLE
Some questions about load_val_bbox()

### ISSUE THREAD

**Issue by xixiobba** [tzzcl/PSOL#4]
hello,I want to know how to modify this file to read the annotations of the validation set during training，Thanks!!!
My file structure is the same as the README
def load_val_bbox(label_dict,all_imgs,gt_location):
    #gt_location ='/data/zhangcl/DDT-code/ImageNet_gt'
    import scipy.io as sio
    gt_label = sio.loadmat(os.path.join(gt_location,'cache_groundtruth.mat'))
    locs = [(x[0].split('/')[-1],x[0],x[1]) for x in all_imgs]
    locs.sort()
    final_bbox_dict = {}
    for i in range(len(locs)):
        #gt_label['rec'][:,1][0][0][0], if multilabel then get length, for final eval
        final_bbox_dict[locs[i][1]] = gt_label['rec'][:,i][0][0][0][0][1][0]
    print(final_bbox_dict)
    return final_bbox_dict

**Comment by xixiobba** [comment]
Another question，why is the validation set used during training？

**Comment by xixiobba** [comment]
Is the accuracy in the paper on the validation dataset? So why does the validation dataset appear during the training process？

**Comment by tzzcl** [comment]
For reading the annotations of the validation set, you can find the file on Google Drive link.

the accuracy in the paper is on the validation set. When we perform training, we load the validation annotation to have a general sense of how that model works, the validation accuracy on the training stage is close to the final evaluation accuracy, but it won't be used in the training stage.

**Comment by xixiobba** [comment]
thanks!!!
